pkg/sql/sem/tree/unsupported.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tree
+
+var _ error = &Unsupported{}
+
+// Unsupported is an error object which is returned by some unimplemented SQL
+// statements. It is currently only used to skip over PGDUMP statements during
+// an import.
+type Unsupported struct {
+	Err                    error
+	SkipDuringImportPGDump bool
```

Comment:
removed the field. I've kept the name as Unsupported since it now backs all unparseable SQL statements. We do only use this fact during an import but I went with a more generic name. Could go either way though.